scene/gui/graph_edit.cpp

```diff
@@ -968,6 +971,11 @@ void GraphEdit::_gui_input(const Ref<InputEvent> &p_ev) {
 		accept_event();
 	}
 
+	if (k.is_valid() && k->get_scancode() == KEY_A && k->get_shift() && k->is_pressed()) {
```

Comment:
you dont need to do this here, you can conenct to the input_event signal of graph_edit,  intercept this key and accept the event

Reply:
Done,  uploading revision.